vulkaninfo/CMakeLists.txt

```diff
@@ -11,7 +11,35 @@ else()
     add_executable(vulkaninfo vulkaninfo.c)
 endif()
 
-if(APPLE)
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    include(FindPkgConfig)
+    option(BUILD_WSI_XCB_SUPPORT "Build XCB WSI support" ON)
+    option(BUILD_WSI_XLIB_SUPPORT "Build Xlib WSI support" ON)
+    set(VULKANINFO_WSI_SELECTION "XCB" CACHE STRING "Select WSI target for vulkaninfo (XCB, XLIB, WAYLAND, MIR, DISPLAY)")
```

Comment:
I think that there are only two choices here that make sense in the STRING parameter.